noir-projects/Dockerfile

```diff
@@ -1,9 +1,12 @@
 FROM aztecprotocol/noir as noir
 FROM aztecprotocol/avm-transpiler as transpiler
+FROM --platform=linux/amd64 aztecprotocol/barretenberg-x86_64-linux-clang as barretenberg
 
 FROM ubuntu:lunar AS builder
 RUN apt-get update && apt-get install -y parallel nodejs npm 
 RUN npm install --global yarn
+# Copy in bb
+COPY --from=barretenberg /usr/src/barretenberg/cpp/build/bin/bb /usr/src/barretenberg/cpp/build/bin/bb
```

Comment:
@ludamad or @spypsy would this have any unintended consequence?

Reply:
this should be fine the only thing is the dependency should be expressed in circleci as well

Reply:
Modified `build-manifest.yml` accordingly!